### DISCUSSION TITLE
Safety Boundaries?

### DISCUSSION THREAD

**Discussion by LoyWeiWin** [visose/Robots#242] [Q&A] [answered]
Hey team,

Just wondering if the current robot plugin would allow the users to establish safety boundaries / safety planes for UR10 via grasshopper? Probably its another silly question.

**Comment by visose** [comment, marked as answer]
You cannot configure this feature from Grasshopper directly, not sure why this would be preferred over configuring it in the controller.
You can of course create a conventional program where the robot moves slower if the target is outside of a certain boundary, but if you need it for safety use the built-in feature.
